Formula/fribidi.rb

```diff
@@ -14,11 +15,19 @@ class Fribidi < Formula
     sha256 "70caed8cb2f44044c41e0b91c2645111b9f177d98b4d49cef01fb0d8558c0f98" => :mavericks
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
   depends_on "pkg-config" => :build
   depends_on "glib"
   depends_on "pcre"
 
   def install
+    # The doc Makefile is broken.
+    # It repeatedly attempts to copy files into the directory they're already in.
+    inreplace "Makefile.am", "doc", ""
```

Comment:
I don't know what to do about this. I'll see how general I can get the failure and report it upstream, I guess.